op-program/host/flags/flags.go

```diff
@@ -83,13 +83,14 @@ var (
 	}
 	L2Claim = &cli.StringFlag{
 		Name:    "l2.claim",
-		Usage:   "Claimed L2 output root to validate",
+		Usage:   "Claimed L2 root to validate",
```

Comment:
Based on https://github.com/ethereum-optimism/optimism/pull/14775, looks like "proposal" is being used to refer to all kinds of L2 outputs roots:
```suggestion
		Usage:   "Claimed proposal root to validate",
```

Though, "proposal" is still a bit ambiguous. It can also refer to the output not just the root. So it helps to be extra explicit with "proposal root".